src/main/java/com/hazelcast/commandline/member/MemberCommandLine.java

```diff
@@ -162,9 +195,12 @@ public void list() {
     }
 
     @Command(description = "Display the logs for Hazelcast member with the given ID.", mixinStandardHelpOptions = true)
-    public void logs(
-            @Parameters(index = "0", paramLabel = "<name>", description = "Unique name of the process to show the logs, for ex.: brave_frog.") String name,
-            @Option(names = {"-n", "--numberOfLines"}, paramLabel = "<lineCount>", description = "Display the specified number of lines (default: 10).", defaultValue = "10") int numberOfLines)
+    public void logs(@Parameters(index = "0", paramLabel = "<name>",
+                                 description = "Unique name of the process to show the logs, for ex"
```

Comment:
That looks like we make it less readable by making description multiline. Can we put such things into one line?

Reply:
Done 👍